### PR TITLE
[Android] Convert ProjectControlsListAdapter class from Java to Kotlin

### PR DESCRIPTION
**Pull request by AenBleidd**
Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

File: android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/ProjectControlsListAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import edu.berkeley.boinc.R
+import edu.berkeley.boinc.rpc.RpcClient
+
+class ProjectControlsListAdapter(
+    context: Context?,
+    // ID of control texts in strings.xml
+    private val entries: List<ProjectControl>
+) : ArrayAdapter<ProjectControl>(
+    context!!, R.layout.projects_controls_listitem_layout, entries
+) {
```
Comment (Isira-Seneviratne) [inline android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/ProjectControlsListAdapter.kt:39]:
```suggestion
class ProjectControlsListAdapter(
    context: Context,
    // ID of control texts in strings.xml
    private val entries: List<ProjectControl>
) : ArrayAdapter<ProjectControl>(
    context, R.layout.projects_controls_listitem_layout, entries
) {
```

Comment (AenBleidd) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/ProjectControlsListAdapter.kt:39]: We're passing nullable type here, so this code fails. I'll keep the original code

Comment (AenBleidd) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/adapter/ProjectControlsListAdapter.kt]: Ok, I made a recheck and fixed it properly

**Review by Isira-Seneviratne — APPROVED** [review]

**Review by talregev — APPROVED** [review]
tested on oneplus one (android 10). it compile run and crunch. added account manger and project. it working.

**Pull request merged**